sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/ecert-full-time-process-integration.scheduler.ts

```diff
@@ -0,0 +1,50 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { ECertFileHandler } from "@sims/integrations/esdc-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { Job, Queue } from "bull";
+import { QueueProcessSummary } from "../../../models/processors.models";
+import { BaseScheduler } from "../../base-scheduler";
+import { ESDCFileResult } from "../models/esdc";
+
+@Processor(QueueNames.FullTimeECertIntegration)
+export class FullTimeECertProcessIntegrationScheduler extends BaseScheduler<void> {
+  constructor(
+    @InjectQueue(QueueNames.FullTimeECertIntegration)
+    schedulerQueue: Queue<void>,
+    queueService: QueueService,
+    private readonly eCertFileHandler: ECertFileHandler,
+  ) {
+    super(schedulerQueue, queueService);
+  }
+
+  /**
+   * Process Full-Time disbursements available to be sent to ESDC.
+   * Consider any record that is scheduled in upcoming days or in the past.
+   * @params job job details.
+   * @returns result of the file upload with the file generated and the
+   * amount of records added to the file.
+   */
+  @Process()
+  async processFullTimeECert(job: Job<void>): Promise<ESDCFileResult> {
+    const summary = new QueueProcessSummary({
+      appLogger: this.logger,
+      jobLogger: job,
+    });
+    await summary.info(
+      `Processing E-Cert fulltime integration job ${job.id} of type ${job.name}.`,
+    );
+    await summary.info("Sending Full-Time E-Cert File...");
+    const uploadFullTimeResult =
+      await this.eCertFileHandler.generateFullTimeECert();
+    await summary.info("E-Cert Full-Time file sent.");
+    await this.cleanSchedulerQueueHistory();
+    await summary.info(
+      `Completed E-Cert fulltime integration job ${job.id} of type ${job.name}.`,
+    );
+    return {
+      generatedFile: uploadFullTimeResult.generatedFile,
```

Comment:
same as https://github.com/bcgov/SIMS/pull/1624/files#r1058660285

Reply:
I believe in your PR, we discussed these new logs changes will be done as a new story, as it is not in the scope of this PR. https://github.com/bcgov/SIMS/pull/1620#discussion_r1053907597.

Reply:
I think we discussed that(changes in new ticket) only in certain places where the `QueueProcessSummaryResult` is not compatible with the current format of data that was being returned. for e.g. SFASProcessingResult[] here. 

I made changes at the verge of PR completion to accomodate QueueProcessSummary or QueueProcessSummaryResult only as applicable.

Reply:
I have created a ticket for it https://app.zenhub.com/workspaces/student-information-management-system-5fce9df5aa1b45000e937014/issues/gh/bcgov/sims/1639

Reply:
I am not in same page. @guru-aot ?

Reply:
Seems okay to me too